### PR TITLE
net-misc/rclone bump 1.49.5

### PR DESCRIPTION
**Pull request by oczkers**
version bump & update github url in other ebuilds

### CODE REVIEW

File: net-misc/rclone/rclone-1.49.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+fi
+
+DESCRIPTION="A program to sync files to and from various cloud storage providers"
+HOMEPAGE="https://rclone.org/"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+DEPEND=""
```
Comment (ConiKost) [inline net-misc/rclone/rclone-1.49.5.ebuild]: If empty, drop it.

File: net-misc/rclone/rclone-1.49.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	SRC_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+	inherit golang-vcs-snapshot
+fi
+
+DESCRIPTION="A program to sync files to and from various cloud storage providers"
+HOMEPAGE="https://rclone.org/"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
```
Comment (ConiKost) [inline net-misc/rclone/rclone-1.49.5.ebuild]: If empty, drop it.

File: net-misc/rclone/rclone-1.49.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (ConiKost) [inline net-misc/rclone/rclone-1.49.5.ebuild]: Is bumping to EAPI=7 possible?

**Comment by oczkers** [comment]
EAPI bumped, empty variables removed and tested

**Comment by oczkers** [comment]
Looks like there are another prs, please consolidate and bump to latest 1.50.1
https://github.com/gentoo/gentoo/pull/13345
https://github.com/gentoo/gentoo/pull/13489

**Pull request closed without merging**